app/views/employees/_form.html.erb

```diff
@@ -1,5 +1,11 @@
 <%= simple_form_for @employee do |form| %>
-  <%= form.input :slack_username %>
+  <%= form.input :slack_username,
+                 required: true,
+                 pattern: '^[^@]\w+[^ ]$',
+                 input_html: {
+                   title: I18n.t('employees.errors.slack_username')
```

Comment:
it seems like if you added this message as a custom message to the validation in the `Employee` class, it should show up if the validation fails

see: http://stackoverflow.com/a/11275441/1019369

I am not sure how to assert validations in a test -- perhaps difficult in which case don't worry about it :cherries: